### DISCUSSION TITLE
Ability to conceal issues and pull requests created in unengaged or inaccessible organizations/repositories

### DISCUSSION THREAD

**Discussion by Xrayez** [community/community#56679] [Repositories]
### Select Topic Area

Product Feedback

### Body

**TL;DR**: a particular organization/repository owner blocked me, but since I previously opened issues and PRs there, I'm unable to close them or hide them from my list of Issues and PRs. It would be nice to hide them. Issues and pull requests that are created by users from organizations that completely block all kinds of interactions with those users serve absolutely no purpose or value because users are unable to take any action on those previously opened issues and submitted PRs.

#### Problem

I have been actively contributing to a specific project for several years and even attained the status of a co-maintainer at one point. Unfortunately, due to unforeseen circumstances, the organization/repository owners decided to permanently ban me from any further interactions with their organization a long ago. For the sake of protecting my image as an individual, I want to clarify that I firmly believe this decision was unjustified, as it was made without valid reasons and potentially impacts my reputation.

Regardless of the reasons behind the ban, it is disheartening to continually encounter all the issues and pull requests I previously opened in an organization/repository that I no longer have access to nor interested in. This situation is particularly demoralizing because this project has been my sole focus for the past few years, and I am now seeking to move on.

Unfortunately, I am unable to close the issues and pull requests I created since the organization/repository blocked me. I have attempted to reach out to the project maintainers, but my requests to close these issues have gone unanswered. It seems that, as a blocked user, my messages are not reaching them, even when I use the officially provided email channels.

#### Suggestion

While I understand that closing all the issues and pull requests may not be a viable solution, as they may still hold relevance for other users, I kindly request the implementation of a feature that allows me to completely hide all issues and pull requests associated with the organization/repository that blocked me. I have not found an analogous feature to block organizations from the standpoint of a user. Only organizations/repositories can block user interactions, and not vice versa. This feature would prevent these items from appearing in my Issues, Pull Requests, and other relevant sections. Since I am unable to take any action on these items, there is no purpose in them being visible to me.

In other words, I want to be able to disconnect myself from a particular organization/repository that no longer serves any purpose or value for me. It's worth noting that I'm not talking about the right to be forgotten, I'm actually interested in retaining all my previous interactions in an organization/repository, so that people can have an opportunity to get insights from my work and conclusions, including conclusions concerning an organization which blocked me, it simply that I want to lift off the baggage of the large number of issues that constitute the majority of my time contributing and being a maintainer of an organization that I'm not interested in anymore.

Thank you for considering my request. Feel free to interpret this request as a feedback though, if that seems more appropriate.
